src/resources/js/tickets.js

```diff
@@ -635,6 +635,14 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 			return;
 		}
 
+		// Does an attendee field validation for multiple labels, if ET+ is active.
+		// Called in event-tickets-plus/src/resources/js/meta-admin.js
+		if ( tribe_event_tickets_plus.meta.admin ) {
```

Comment:
`tribe_event_tickets_plus` won't exist if ET+ isn't active, so this will throw an error, I think.

Probably need to check for it and the meta property, first.
```suggestion
		if ( typeof tribe_event_tickets_plus !== 'undefined' && tribe_event_tickets_plus.meta && tribe_event_tickets_plus.meta.admin ) {
```

Reply:
ended up removing this for a different solution Juan recommended